src/main/java/com/payw/npoem/domain/entry/PoemRepository.java

```diff
@@ -0,0 +1,6 @@
+package com.payw.npoem.domain.entry;
```

Comment:
Is this just for a unique identification? Doesn't it matter we are not owning `payw.com`?

If it matters, it should be refactored to `org.payw.~`.

Reply:
i changed package name